src/ApplicationManager.cpp

```diff
@@ -160,33 +160,45 @@ void ApplicationManager::runOneLoopCycle() {
 }
 
 void ApplicationManager::decidePlay(world::World *_world) {
```

Comment:
Is it checked before decidePlay is called if _world has a value? Otherwise possible segfault problem in playevaluator.update(_world) (line 168)